### PR TITLE
🚀 Debezium 1.0 release announcement

### PR DESCRIPTION
**Pull request by gunnarmorling**
@jpechane, @Naros, @jcechace, any feedback welcome. I'm still adding a few more links and a visual for a data streaming pipeline to the use cases section, but the text is mostly done. Let me know about your feedback. Thanks!

### CODE REVIEW

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+gmorling
+:awestruct-tags: [ releases, mysql, postgres, sqlserver, mongodb, cassandra, oracle ]
+:awestruct-layout: blog-post
+
+Today it's my great pleasure to announce the availability of Debezium *1.0.0.Final*!
+
+Since the https://github.com/debezium/debezium/commit/0a99ed67cd8f74d6f451b0a2d3809e23127e4698[initial commit] in November 2015,
+the Debezium community has worked tirelessly to realize the vision of building a comprehensive open-source low-latency platform for change data capture for a variety of databases.
+
+Within those four years, Debezium's feature set has tremendously grown: stable, highly configurable CDC connectors for MySQL, Postgres, MongoDB and SQL Server, incubating connectors for Apache Cassandra and Oracle, facilities for transforming and routing messages, support for design patterns such as the outbox pattern and much more.
```
Comment (sjwoodman) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: "grown tremendously"

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+A very active, open and welcoming community of users, contributors and committers has formed around the project.
+Debezium is deployed to production at lots of organizations from all kinds of industries,
+partly with huge installations, using hundreds of connectors to stream data changes out of thousands of databases.
```
Comment (sjwoodman) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: some with huge installations

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+When using Debezium with Apache Kafka and its rich ecosystem of sink connectors
+setting up such integrations can be done without any coding,
+just by means of configuration.
+
+But there's many other use cases of CDC which go beyond just moving data from A to B.
+When adding stream processing into the picture, e.g. via Kafka Streams or Apache Flink,
+CDC enables you to run https://github.com/debezium/debezium-examples/tree/master/kstreams-live-update[time-windowed streaming queries], continuously updated as your data changes
+("what's the aggregated order revenue per category within the last hour").
+You can use CDC to link:/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[build audit logs] of your data,
+telling who did change which data items at which time.
```
Comment (sjwoodman) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: telling who changed which data items at what time

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+== Debezium 1.0
+
+Now, let's talk a little bit about the contents of the 1.0 release.
+
+This version continues the efforts we've begun in link:/blog/2019/10/02/debezium-0-10-0-final-released/[0.10] to make sure the emitted event structures and configuration options of the connectors are correct and consistent.
+While we've always been very careful to ensure a smooth upgrading experience, you can expect even more stability in this regard going forward after the 1.0 release.
+
+We've expanded the test coverage of databases (Postgres 12, SQL Server 2019),
+build against the latest version of Apache Kafka (2.4.0) and upgraded our container images to OpenJDK 11.
```
Comment (sjwoodman) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: build -> built

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+You're amazing, and I would like to wholeheartedly thank each and everyone of you!
+I'm sure our community is continuing to grow in the future -- I'd love if we hit the mark of 200 contributors in 2020.
+
+Equally important are our users; interacting with you in the chat, on the mailing list or at conferences and meet-ups is what helps to drive the direction of the project:
+learning about your specific requirements and use cases (or bugs you've run into) is vital for deciding where to put the focus next.
+A big thank you to you, too!
+
+Some of you even have shared their experiences with Debezium in conference talks and blog posts.
+Nothing beats to hear the war stories from others and being able to learn from their experiences,
```
Comment (sjwoodman) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: "Nothing beats hearing the war stories shared by others"

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+* Replicating data from production databases to other databases and data warehouses
+* Feeding data to search services like Elasticsearch or Apache Solr
+* Updating or invalidating caches
+
+When using Debezium with Apache Kafka and its rich ecosystem of sink connectors
+setting up such integrations can be done without any coding,
+just by means of configuration.
+
+But there's many other use cases of CDC which go beyond just moving data from A to B.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
But there are many other use cases of CDC which go beyond just moving data from A to B.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 3 above)
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
telling who changed which data items at what time.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+When adding stream processing into the picture, e.g. via Kafka Streams or Apache Flink,
+CDC enables you to run https://github.com/debezium/debezium-examples/tree/master/kstreams-live-update[time-windowed streaming queries], continuously updated as your data changes
+("what's the aggregated order revenue per category within the last hour").
+You can use CDC to link:/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[build audit logs] of your data,
+telling who did change which data items at which time.
+Or update denormalized views of your data, for the sake of efficient data retrieval, adhering to the CQRS pattern.
+
+Finally, CDC also can play a vital role in microservices architectures;
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
Finally, CDC can also play a vital role in microservices architectures;
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+exchanging data between services and keeping local views of data owned by other services achieves a higher independence, without having to rely on synchronous API calls.
+One particularly interesting approach in this context is the link:/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[outbox pattern],
+which is link:/documentation/reference/1.0/configuration/outbox-event-router.html[supported well] by Debezium.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
which is link:/documentation/reference/1.0/configuration/outbox-event-router.html[well supported] by Debezium.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+In case you don't start on the green field (who ever does?),
+CDC can be used to implement the strangler pattern for moving from a monolithic design to microservices.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: link to something describing this pattern?

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+setting up such integrations can be done without any coding,
+just by means of configuration.
+
+But there's many other use cases of CDC which go beyond just moving data from A to B.
+When adding stream processing into the picture, e.g. via Kafka Streams or Apache Flink,
+CDC enables you to run https://github.com/debezium/debezium-examples/tree/master/kstreams-live-update[time-windowed streaming queries], continuously updated as your data changes
+("what's the aggregated order revenue per category within the last hour").
+You can use CDC to link:/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[build audit logs] of your data,
+telling who did change which data items at which time.
+Or update denormalized views of your data, for the sake of efficient data retrieval, adhering to the CQRS pattern.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Should you define CQRS?

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+partly with huge installations, using hundreds of connectors to stream data changes out of thousands of databases.
+
+The 1.0 release marks an important milestone for the project:
+based on all the production feedback we got from the users of the 0.x versions, we figured it's about time to express the maturity of the four stable connectors in the version number, too.
+
+== Why Debezium?
+
+One of the things making it so enjoyable to work on Debezium as a tool for change data capture is the variety of potential use cases.
+When presenting the project at conferences,
+it's just great to see how people quickly get excited when they realize all the possibilities enabled by Debezium and CDC.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Define CDC?

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+you can find lots of blog posts, conference talks and examples by folks using Debezium in production in our compilation of link:/documentation/online-resources/[resources].
+If you'd like to get a glimpse of who else already is using Debezium,
+see our rapidly growing link:/community/users/[list of reference users]
+(or send us a https://github.com/debezium/debezium.github.io/blob/develop/community/users.asciidoc[pull request] to get your name added if your organization already is running Debezium in production).
+
+== Debezium 1.0
+
+Now, let's talk a little bit about the contents of the 1.0 release.
+
+This version continues the efforts we've begun in link:/blog/2019/10/02/debezium-0-10-0-final-released/[0.10] to make sure the emitted event structures and configuration options of the connectors are correct and consistent.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
This version continues the effort we began in link:/blog/2019/10/02/debezium-0-10-0-final-released/[0.10] to make sure the emitted event structures and configuration options of the connectors are correct and consistent.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 4 above)
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: The present tense of "build" conflicts with the past tense of "expanded" and "upgraded". But using "built" doesn't really work either. So how about 

We've expanded the test coverage of databases (Postgres 12, SQL Server 2019),
upgraded our container images to OpenJDK 11 and we build against the latest version of Apache Kafka (2.4.0).

which sounds better to me, even if not perfect.

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+
+You're amazing, and I would like to wholeheartedly thank each and everyone of you!
+I'm sure our community is continuing to grow in the future -- I'd love if we hit the mark of 200 contributors in 2020.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
I'm sure our community will continue to grow in the future -- I'd love it if we hit the mark of 200 contributors in 2020.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+
+Equally important are our users; interacting with you in the chat, on the mailing list or at conferences and meet-ups is what helps to drive the direction of the project:
+learning about your specific requirements and use cases (or bugs you've run into) is vital for deciding where to put the focus next.
+A big thank you to you, too!
+
+Some of you even have shared their experiences with Debezium in conference talks and blog posts.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
Some of you have even shared their experiences with Debezium in conference talks and blog posts.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 5 above)
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
Nothing beats hearing the war stories of others and being able to learn from their experiences,
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+Nothing beats to hear the war stories from others and being able to learn from their experiences,
+so you speaking about your insights around Debezium and CDC is incredibly helpful and highly appreciated!
+
+== What's Next?
+
+Let's wrap up this post with an outlook to what's next in store for Debezium.
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]:
```suggestion
Let's wrap up this post with a look to see what's next in store for Debezium.
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+We also have some exciting blog posts in the workings,
+e.g. on how to combine Debezium with the brand-new Kafka Connect https://camel.apache.org/blog/Camel-Kafka-connector-intro/[connector for Apache Camel] or how to use the recently added support for non-key joins in Kafka Streams (https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=74684836[KIP-213]) with Debezium change events.
+
+One more thing I'm super-thrilled about is Debezium becoming a supported component of the https://www.redhat.com/en/blog/whats-new-red-hat-integration[Red Hat Integration] product.
+Part of the current release is a https://developers.redhat.com/blog/2019/11/22/red-hat-advances-debezium-cdc-connectors-for-apache-kafka-support-to-technical-preview/[Tech Preview] for the change data capture connectors for MySQL, Postgres, SQL Server and MongoDB.
+This is great news for folks who wish to have commercial support by Red Hat for their CDC connectors.
+
+For now, let's celebrate the release of Debezium 1.0 and look forward to what's coming in 2020.
+
+Upwards and Onwards!
```
Comment (tombentley) [inline blog/2019-12-18-debezium-1-0-0-final-released.adoc]: It's usually the other way around:

```suggestion
Onwards and Upwards!
```

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Funny, it seems some of my mistakes sound so wrong to the native speaker's ear, that both you and @sjwoodman pointed them out :)

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Went for Simon's "some with".

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 11 above)
Comment (gunnarmorling) [reply blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Good point. I've changed the first occurence in the post to "change data capture (CDC)".

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 10 above)
Comment (gunnarmorling) [reply blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Done.

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 9 above)
Comment (gunnarmorling) [reply blog/2019-12-18-debezium-1-0-0-final-released.adoc]: 👍  Done.

File: blog/2019-12-18-debezium-1-0-0-final-released.adoc (hunk 4 above)
Comment (gunnarmorling) [reply blog/2019-12-18-debezium-1-0-0-final-released.adoc]: Thx, changed it along these lines.

**Comment by gunnarmorling** [comment]
Thanks a lot for reviewing, @sjwoodman and @tombentley. Addressed all your remarks.

**Pull request merged**